update.sh

```diff
@@ -29,24 +29,39 @@ for version in "${versions[@]}"; do
 
 	echo "$version: $fullVersion (ruby $rubyVersion; passenger $passenger)"
 
+	sedExpr='
+			s/%%REDMINE_VERSION%%/'"$fullVersion"'/;
+			s/%%RUBY_VERSION%%/'"$rubyVersion"'/;
+			s/%%REDMINE_DOWNLOAD_MD5%%/'"$md5"'/;
+			s/%%REDMINE%%/redmine:'"$version"'/;
+			s/%%PASSENGER_VERSION%%/'"$passenger"'/;
+		'
+	sedAlpineExpr="$sedExpr"
+	sedDebianExpr="$sedExpr"
+
+	if [ "$version" = 3.4 ] || [ "$version" = 4.0 ]; then
+		sedAlpineExpr+='
+			s/imagemagick /imagemagick6 /;
+			/ghostscript /d;
+			/gcc/a \\t\timagemagick6-dev \\
+		'
+		sedDebianExpr+='
+			/ghostscript /d;
+			/gcc/a \\t\tlibmagickcore-dev \\
+			/gcc/a \\t\tlibmagickwand-dev \\
+		'
+	fi
```

Comment:
I don't love how specific to the current template contents this ends up being -- is there any downside to just including `ghostscript` all the time and just removing the `-dev` packages in 4.1+? (swapping from `imagemagick` to `imagemagick6` in Alpine)

I think I'd rather see this implemented as a Bash array too, as in:

```bash
commonSedArgs=(
	-e 's/%%REDMINE_VERSION%%/'"$fullVersion"'/'
	-e 's/%%RUBY_VERSION%%/'"$rubyVersion"'/'
	-e 's/%%REDMINE_DOWNLOAD_MD5%%/'"$md5"'/'
)
alpineSedArgs=()

...

alpineSedArgs+=( -e 's/imagemagick/imagemagick6/g' )

...

sed -r "${commonSedArgs[@]}" "${debianSedArgs[@]}" Dockerfile-debian.template > "$version/Dockerfile"
...
```

Reply:
`ghostscript` is quite large and the processing of PDFs is somewhat risky. I'd prefer to have it not installed if not required.

Reply:
Makes sense -- I'll take a stab at adjusting `update.sh` and push a commit here for you to review. :+1:

(And thanks for pointing out that I was being silly in #189 :sweat_smile: -- still coming out from holiday backlog which I'll probably be feeling like all the way until the holiday season again this year :face_with_head_bandage:)